beacon-chain/forkchoice/protoarray/store.go

```diff
@@ -1052,3 +1069,23 @@ func (f *ForkChoice) JustifiedPayloadBlockHash() [32]byte {
 	node := f.store.nodes[idx]
 	return node.payloadHash
 }
+
+// HighestReceivedBlockSlot returns the highest slot received by the forkchoice
+func (f *ForkChoice) HighestReceivedBlockSlot() types.Slot {
+	f.store.nodesLock.RLock()
+	defer f.store.nodesLock.RUnlock()
+	return f.store.highestReceivedSlot
+}
+
+// ReceivedBlocksLastEpoch returns the number of blocks received in the last epoch
+func (f *ForkChoice) ReceivedBlocksLastEpoch() uint64 {
+	f.store.nodesLock.RLock()
+	defer f.store.nodesLock.RUnlock()
+	count := uint64(0)
+	for _, s := range f.store.receivedBlocksLastEpoch {
+		if s != 0 {
+			count++
+		}
```

Comment:
Another option is to add a separate counter variable, so we don't need this loop. Happy to do that as well